Open Judge System/Services/OJS.Services/Business/ParticipantScores/Models/ContestParticipationSummaryExcel.cs

```diff
@@ -0,0 +1,155 @@
+﻿using System;
+
+namespace OJS.Services.Business.ParticipantScores.Models
+{
+    using System.IO;
+    using System.Linq;
+    using MissingFeatures;
+    using NPOI.HSSF.UserModel;
+    using NPOI.SS.UserModel;
+    using System.Collections.Generic;
+    using OJS.Web.Common.Extensions;
+    using OJS.Common.Extensions;
+    
+    public class ContestParticipationSummaryExcel
+    {
+        public HSSFWorkbook Workbook { get; set; }
+        
+        public ContestParticipationSummary ContestParticipationSummary { get; set; }
+
+        private int rowNumber = 1;
+        private int preProblemsColumnsCount = 4;
+        private ISheet sheet;
+
+        public ContestParticipationSummaryExcel(ContestParticipationSummary summary)
+        {
+            this.ContestParticipationSummary = summary;
+            this.Create();
+        }
+
+        public MemoryStream GetAsStream()
+        {
+            var outputStream = new MemoryStream();
+            Workbook.Write(outputStream);
+            
+            return outputStream;
+        }
```

Comment:
The returned stream is not disposed. Should be wrapped into using and return byte[] instead of the stream directly or should be wrapped into using in StatisticsController.cs GetContestParticipantScoreInfo